qa/tasks/mgr/dashboard/test_health.py

```diff
@@ -89,19 +89,22 @@ def test_full_health(self):
             'df': JObj({
                 'pools': JList(JObj({
                     'stats': JObj({
-                        'wr': int,
-                        'quota_objects': int,
+                        'stored': int,
+                        'objects': int,
+                        'kb_used': int,
                         'bytes_used': int,
+                        'percent_used': float,
                         'max_avail': int,
+                        'quota_objects': int,
+                        'quota_bytes': int,
+                        'dirty': int
```

Comment:
missing a `,`.
```
invalid syntax (test_health.py, line 101)
```
see http://pulpito.ceph.com/kchai-2018-12-02_07:20:25-rados-wip-kefu2-testing-2018-12-02-0912-distro-basic-smithi/3302833/

Reply:
@tchaikov - thanks for pointing this out, fixed!